src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,108 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private static User john;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        john = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        john.setAge(19);
+        john.setLogin("John12.03.1993");
+        john.setPassword("12345678");
+    }
+
+    @Test
+    void validUser_Ok() {
+        registrationService.register(john);

```

Comment:
lets also check is user that returned from register method is the same as we want to register(have the same fields)